examples/twri/scripts/BGCompass_inversion_tti.jl

```diff
@@ -0,0 +1,92 @@
+################################################################################
+#
+# Run inversion for the BG Compass model
+#
+################################################################################
+
+### Module loading
+using LinearAlgebra, Dates, Distributed, Images, Random
+using JUDI, JUDI.TimeModeling, JUDI.SLIM_optim, JLD2
+
+R = Float32
+### Load synthetic data
+base_path = dirname(pathof(JUDI))*"/../examples/twri/"
+@load string(base_path*"data/BGCompass_data_tti.jld") model_true dat fsrc
+
+# Different sigmas for smoothing
+vvar = [20, 25, 30, 40]
+
+# Bound projection
+function proj_bounds(m, mmin, mmax)
+    m[m[:] .< mmin] .= mmin
+    m[m[:] .> mmax] .= mmax
+    return m
+end
+mmin = 1f0/4.5f0^2
+mmax = 1f0/1.4f0^2
+ProjBound(x) = proj_bounds(x, mmin,  mmax)
+
+for var=vvar
+    for smooth_thomsen=[true, false]
```

Comment:
Seems like `smooth_thomsen=[true, false]` can be replaced by `anis = ["st", "tt"]`

Reply:
good point, done